src/Http/Wolverine.Http.Tests/endpoint_adds_requesttype_audit_tags_to_activity.cs

```diff
@@ -0,0 +1,29 @@
+using JasperFx.Core;
+using Shouldly;
+using WolverineWebApi;
+
+namespace Wolverine.Http.Tests;
+
+public class endpoint_adds_requesttype_audit_tags_to_activity : IntegrationContext
+{
+    public endpoint_adds_requesttype_audit_tags_to_activity(AppFixture fixture) : base(fixture)
+    {
+    }
+
+    [Fact]
+    public void finds_audit_members_from_attributes()
+    {
+        var chain = HttpChains.ChainFor("POST", "/auditable/empty");
+        
+        chain.AuditedMembers.Single()
+            .MemberName.ShouldBe(nameof(AuditablePostBody.Id));
+    }
+    
+    [Fact]
+    public void adds_auditable_members_to_activity()
+    {
+        var chain = HttpChains.ChainFor("POST", "/auditable/post");
+        var lines = chain.SourceCode.ReadLines();
```

Comment:
This test currently fails because I can't get the `SourceCode` to be initialized.